src/test/java/net/sf/jabref/importer/fileformat/PdfContentImporterTestFiles.java

```diff
@@ -0,0 +1,57 @@
+package net.sf.jabref.importer.fileformat;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.bibtex.BibtexEntryAssert;
+import net.sf.jabref.model.entry.BibEntry;
+
+@RunWith(Parameterized.class)
+public class PdfContentImporterTestFiles {
+
+    @BeforeClass
+    public static void setUp() {
+        Globals.prefs = JabRefPreferences.getInstance();
+    }
+
+    @Parameters(name = "{index}: {0}")
+    public static Collection<Object[]> fileNames() {
+        // The test folder contains pairs of PDFs and BibTeX files. We check each pair.
+        // This method returns the basenames of the available pairs
+
+        // currently, only one test case "LNCS-minimal" is existing
+        // code review demanded that @Paramters should be used
+        // Arrays.asList only works for multiple parameters, therefore, there are two parameters written
+        return Arrays.asList(new Object[] {"LNCS-minimal"}, new Object[] {"LNCS-minimal"});
```

Comment:
You can use something like here https://github.com/uniba-dsg/BPELlint/blob/master/src/test/java/bpellint/SAViolationTestData.java which allows for a much nicer structure of the input paramters.